packages/config/src/types.ts

```diff
@@ -152,14 +152,52 @@ export interface ProjectLinks {
   socialMedia?: string[]
   rollupCodes?: string
 }
-
-export interface Badge {
+export type Badge = {
```

Comment:
why the switch from interface to type? Everything else in this file is interfaces